jq/script-03.sh

```diff
@@ -5,3 +5,5 @@ set -euo pipefail
 # The input for this script is the person.json file.
 # TODO: Write a command to output the name of the person, then a comma, then their profession.
 # Your output should be exactly the string "Selma, Software Engineer", but should not contain any quote characters.
+q -r '[.name, .profession] | join (",")' person.json
```

Comment:
It looks like something went slightly wrong in your copy+paste here?

But this raises an interesting workflow question - there are two ways you can go about making this kind of script - you can work in the terminal then copy+paste into the file, or work in the file and run the file each time you want to try something out. How do you feel about the two? What are the advantages/disadvantages of each way of working?